portlist/clean.go

```diff
@@ -24,6 +24,12 @@ func argvSubject(argv ...string) string {
 	case ret == "mono" && len(argv) >= 2:
 		// .Net programs execute as `mono actualProgram.exe`.
 		ret = filepath.Base(argv[1])
+	case strings.Contains(ret, " "):
+		argv2 := strings.Split(ret, " ")
```

Comment:
Split allocates. Use two strings.Cut instead?

Reply:
Oh, I could put it straight into `ret`, I suppose it will always be found.